src/test/gather_node_test.cpp

```diff
@@ -174,8 +263,8 @@ TEST_F(GatherNodeTest, FullFlowGatherInNonExitNode) {
     oneDummyNodeSessionResults1.insert({subsessions[0].getSessionKey(), {subsessions[0], dummy1Result}});
     oneDummyNodeSessionResults2.insert({subsessions[1].getSessionKey(), {subsessions[1], dummy2Result}});
     // actual test steps
-    gather2DummyNode.setInputs(oneDummyNode1, oneDummyNodeSessionResults1);
-    gather2DummyNode.setInputs(oneDummyNode1, oneDummyNodeSessionResults2);
+    ASSERT_TRUE(gather2DummyNode.setInputs(oneDummyNode1, oneDummyNodeSessionResults1).ok());
```

Comment:
When expecting TRUE from ok() call you do not get detailed information about actual response code in case of error. We always do:
`ASSERT_EQ(gather2DummyNode.setInputs(oneDummyNode1, oneDummyNodeSessionResults1), StatusCode::OK);`

Reply:
This will print expected status code if tests failed